### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by vikivivi** [comment]
`predicable` seem to be a valid word in hunspell, aspell, ispell, etc.
But it may not found in all online dictionaries.

https://www.merriam-webster.com/dictionary/predicable
https://www.collinsdictionary.com/dictionary/english/predicable
https://en.wikipedia.org/wiki/Predicable
https://wikidiff.com/predictable/predicable
https://www.quora.com/Is-predicable-interchangeable-with-predictable

**Comment by luzpaz** [comment]
Suggestions on how to proceed ?

**Comment by vikivivi** [comment]
I suggest moving `predicable->predictable` to dictionary_rare.txt

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -29012,6 +29041,8 @@ speciic->specific
 speciied->specified
 speciifc->specific
 speciifed->specified
+specilization->specialization
+specilizations->specializations
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:29045]: Can you separately add the `isa` versions for BE.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -21363,6 +21386,7 @@ nuissance->nuisance
 nulk->null
 Nullabour->Nullarbor
 nulll->null
+numbe->number
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:21389]:
```suggestion
numbe->number, numb,
```

**Review by peternewman — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:29068]: Added in 6af8b7f

**Comment by luzpaz** [comment]
> I suggest moving `predicable->predictable` to dictionary_rare.txt 

Done in 31f4c3c

**Pull request merged**